src/IIIFPresentation/API.Tests/Integration/ModifyManifestUpdateTests.cs

```diff
@@ -537,4 +537,32 @@ public async Task PutFlatId_Update_IgnoresPaintedResources_WhenItemsAndPaintedRe
         errorResponse!.Detail.Should().Be("Could not deserialize manifest");
         errorResponse.ErrorTypeUri.Should().Be("http://localhost/errors/ModifyCollectionType/CannotDeserialize");
     }
+    
+    [Fact]
+    public async Task UpdateManifest_BadRequest_WhenManifestWithBatchWithoutAssets()
+    {
+        // Arrange
+        var createdDate = DateTime.UtcNow.AddDays(-1);
```

Comment:
Tiny: `createdDate`, `slug` and `manifest` are all superfluous to the test and don't have any bearing on the result. Remove to keep only relevant test data in scope?